source/custom_templates/_tx_api_page.txt

```diff
@@ -1,8 +1,9 @@
 .. include:: _:page:.tmp
 
-.. include:: common_response_params.rst
+.. include:: _:org_type:_param_table.rst
 
-.. include:: optional_fields.txt
+The following fields are optional based on your account type:
 
+.. include:: _:org_type:_optional_param_table.rst
 
```

Comment:
@nburwell Do I have these right? 
1. Call center integration params are included in Network reports by default (and are therefore not optional to Networks)? 

2. Signal params are never available to Affiliates and sort of optional for Network and Advertiser? (Looking at the DetailReport code though it seems that the majority of customers would get signal data out of the box, our dummy users in the scripts got Signal params from static columns not dynamic).

If the above is correct, it's going to be a little more work to try and make this DRY since all 3 org_types use the same template. I can make it work with a little massaging though.